db/migrate/20200916100344_create_events.rb

```diff
@@ -0,0 +1,9 @@
+rclass CreateEvents < ActiveRecord::Migration[6.0]
```

Comment:
# Fix the typo error
- [ ] Remove the unnecessary character at the beginning of the `CreateEvents` class definition. It is causing migration errors for users of your code, thank you!
```suggestion
class CreateEvents < ActiveRecord::Migration[6.0]
```